### ISSUE TITLE
Nicer emails

### ISSUE THREAD

**Issue by fellhorn** [studieren-ohne-grenzen/schlemmenohnegrenzen#3]
HTML mails and also confirmation of user input data

**Comment by fellhorn** [comment]
We should also add some basic information about the event:
Date & time when it starts, timeslots of the dishes, ...

**Comment by marcelklehr** [comment]
Where in the code base is the code that sends mails?

**Comment by marcelklehr** [comment]
Nevermind. grep ftw! ;)

**Comment by marcelklehr** [comment]
Is there a reason the confirmation email is not sent to both participants of a house in one go?

**Comment by danielrenninghoff** [comment]
No :D. You can change that. But what I will do is send the sepa lastschriftmandat only to email1 (in a separate mail).

**Comment by danielrenninghoff** [comment]
Actually, we use the name of the person in the email-text, so it needs to be separate.

**Comment by marcelklehr** [comment]
Great. We could write "Hello {name1} and {name2}", though.

**Comment by danielrenninghoff** [comment]
Ok, that sounds like a good idea :)! btw, I just commited a change to the email_senden function.

**Comment by marcelklehr** [comment]
In a68cd76d16ac30ea1cab72094010defeaf3e4d60...master I introduced mail templates. I haven't tested actually sending the mail as I don't have a mail server, though.

**Comment by marcelklehr** [comment]
Also, The about-us section would benefit from being placed into a separate, reusable template, so both mails and frontend can use it...

**Comment by danielrenninghoff** [comment]
The problem with html emails is that you can't be sure how they will get rendered and that is a big problem. A lot of email clients don't load external sources by default and then your emails will look really bad. Another problem is HTML5 support. The current templates you developed use bootstrap and HTML5. Most email clients don't even render divs correctly and that is why email templates often still use tables and inline styles.

For more information:
https://stackoverflow.com/questions/127498/what-guidelines-for-html-email-design-are-there
https://stackoverflow.com/questions/4829254/best-practices-for-styling-html-emails
http://foundation.zurb.com/emails.html

The last link shows an alternative to bootstrap that is specifically optimized for emails.

So in total, crafting email templates is a complicated process that imho is not worth it for our use case. It is especially not feasible to share code between the main site and the html template.

I'm of the opinion that we should remove the html mails in their current form, because having a simple plaintext email that everybody can read is better than having a html email that looks like **** because the client is rendering it wrong.

On the other hand I think your plaintext templates are really nice and we should continue using them.

Any opinions about this topic?

**Comment by marcelklehr** [comment]
Fuu, didn't know about that... That sucks. Then I agree that we should avoid HTML mails.

**Comment by fellhorn** [comment]
Yeah I know about this issue. I used the zurb templates for a customer once. They provide something like a very basic common understanding for almost all mail clients. I personally like the welcome template from this site:
http://foundation.zurb.com/emails/email-templates.html

We should definitely spend not more effort than using a copy & paste template. In my opinion it would be already enough using only some table layouting and putting the logo in the mail. Just to make it a bit nicer. (Says the guy reading mails only in plaintext...)

**Comment by marcelklehr** [comment]
Do we need to confirm emails, btw, to make sure the people actually
signed up?
